packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme.ts

```diff
@@ -80,6 +80,7 @@ const paymentLightTheme: Theme = {
         },
       },
     },
+    overlay: { background: globalColors.neutral.blueGrayLight.a200 },
```

Comment:
Why is prettier not moving this to a new line? 😰
```suggestion
    overlay: { 
    	background: globalColors.neutral.blueGrayLight.a200 
    	},
```

Reply:
we need to manually move it for some cases 🤕

Reply:
So much better now ❤️